src/main/resources/default/taglib/w/multiLanguageTextField.html.pasta

```diff
@@ -88,7 +88,7 @@
                 fallbackLabel: '@i18n("Language.fallback")',
                 wrapperId: '___fieldId',
                 modalId: '___modalId',
-                values: JSON.parse(<i:raw>'___value.getAsJSON()'</i:raw>),
+                values: JSON.parse(<i:raw>'@escapeJS(value.getAsJSON())'</i:raw>),
```

Comment:
Just out of curiosity, why doesn't:
`values: <i:raw>@value.getAsJSON()</i:raw>`
work?